neuraxle/union.py

```diff
@@ -97,6 +97,27 @@ def transform(self, data_inputs):
         results = self.joiner.transform(results)
         return results
 
+    def inverse_transform(self, processed_outputs):
+        """
+        Inverse Transform the data with the unions. It will make use of some parallel processing.
+
+        :param processed_outputs: The input data to inverse transform onto
+        :return: the transformed data_inputs.
+        """
+        if self.n_jobs != 1:
+            results = Parallel(backend=self.backend, n_jobs=self.n_jobs)(
+                delayed(bro.inverse_transform)(processed_outputs)
```

Comment:
this won't work. The joiner needs to inverse transform and that's something quite hard to do because it'd need to remember the number of things joined somehow and hence build a cache in itself.